Makefile.target

```diff
@@ -135,7 +135,7 @@ obj-y += memory.o savevm.o cputlb.o
 obj-y += memory_mapping.o
 obj-y += dump.o
 obj-$(call lnot,$(CONFIG_ROCKER)) += hw/net/rocker/qmp-norocker.o
-LIBS+=$(libs_softmmu)
+LIBS+=$(libs_softmmu) -lJudy
```

Comment:
If the answer to "can ghash be used here instead of libjudy" is yes, then you probably need to convert to ghash.  I would expect push-back from qemu community on adding libjudy and this is the first question they'll ask.